lang/vi.js

```diff
@@ -0,0 +1,220 @@
+export default {
+	"components.alert.close": "Đóng Cảnh Báo",
+	"components.breadcrumbs.breadcrumb": "Breadcrumb",
```

Comment:
This term should have been translated. Requires circling back to localization team.